### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by ericpre**
Align installation instructions with lastest changes of the HyperSpy bundle.

### CODE REVIEW

File: doc/user_guide/install.rst [hunk 1]
```diff
@@ -67,18 +68,28 @@ manager:
...
 
-
    .. code-block:: bash
 
        $ conda install hyperspy-base -c conda-forge
 
+.. note::
+
+    Depending on how Anaconda have been installed, it is possible that the 
+    ``conda`` is not avaible from the Terminal, read the
```
Comment (jlaehne) [inline doc/user_guide/install.rst]: ... the ``conda`` *command* is

File: doc/user_guide/install.rst [hunk 2]
```diff
@@ -27,38 +30,36 @@ To enable context-menu (right-click) shortcut in a chosen folder, use the
...
-Bundle. This is a customised `WinPython <http://winpython.github.io/>`_
-distribution that includes HyperSpy, all its dependencies and many other
-scientific Python packages.
+For instructions and download links go to https://github.com/hyperspy/hyperspy-bundle
+
+Portable distribution (Windows only)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A portable version of the `HyperSpy bundle <https://github.com/hyperspy/hyperspy-bundle>`_
+based the WinPython distribution is also available on Windows.
```
Comment (jlaehne) [inline doc/user_guide/install.rst]: ... based *on* the WinPython ...

File: doc/user_guide/install.rst [hunk 3]
```diff
@@ -67,18 +68,28 @@ manager:
...
 
+.. note::
+
+    Depending on how Anaconda have been installed, it is possible that the 
+    ``conda`` is not avaible from the Terminal, read the
+    `Anaconda User Guide <https://docs.continuum.io/anaconda/>`_ for details.
+
 .. note::
 
     Using ``-c conda-forge`` is only necessary when the conda-forge is not
```
Comment (jlaehne) [inline doc/user_guide/install.rst]: ... when conda-forge ...

File: doc/user_guide/install.rst [hunk 4]
```diff
@@ -67,18 +68,28 @@ manager:
...
     for more details.
 
+.. note::
+
+    Depending on the packages installed in Anaconda, ``conda`` can be slow and
+    in this case ``mamba`` can be used as replacement of ``conda``. Read the
+    `mamba documentation <https://github.com/mamba-org/mamba>`_ for instructions.
 
 Further information
 ^^^^^^^^^^^^^^^^^^^
```
Comment (jlaehne) [inline doc/user_guide/install.rst:96]: a solution for dependencies resolution -> dependencies
In such situation -> In such a situation

**Review by jlaehne — APPROVED** [review]
A few small corrections, otherwise LGTM

**Comment by ericpre** [comment]
Thanks @jlaehne!

**Comment by francisco-dlp** [comment]
What would you think about mentioning Miniforge in addition to Miniconda or maybe even instead of Miniconda? That's what I am using after some headaches with mixing package from the default and conda forge channels.

**Comment by ericpre** [comment]
> What would you think about mentioning Miniforge in addition to Miniconda or maybe even instead of Miniconda? That's what I am using after some headaches with mixing package from the default and conda forge channels.

Miniforge *on windows* is very very recent and may need to mature a bit in my opinion! For the moment, I think we should mention Anaconda/Miniconda only and revisit later?

**Pull request merged**